recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
IMHO go for the Python package name (i.e. `av`). Many often assume `conda` is like `pip`. So it is good if we can match `pip`'s behavior.